src/ex3/index.js

```diff
@@ -0,0 +1,75 @@
+import { Command } from "commander";
+import { myLogger } from "./logger/logger.js";
+import { readFile,writeFile,unlink } from 'fs';
+import ItemManager from "./ItemManager.js";
+const myTasks = new ItemManager();
+
+function getCommanderProgram() {
+  const program = new Command();
+
+  program
+    .name("todo-app")
+    .description("Use the todo app to make to-do list + catch pokemons!")
+    .version("1.0.0");
+
+  program
+    .command("add")
+    .description("Add to-do to the list")
+    .argument("<string>", "your TO-DO")
+    .action(async (task) => {
+      const tasks = await (myTasks.add(task));
+
+      console.log(`New to-do add successfully `+JSON.stringify(tasks));
+      tasks.map(value => myLogger.log(value));
+    });
+
+    program
+    .command("get")
+    .description("Get the to-do list")
+    .action(async () => {
+      readFile('./log.txt', 'utf8', (err, data) => {
```

Comment:
You should have used a JSON file as it is more common and fits better for this case. TXT file is not structured and might lead to errors in the future